Recrutify/Recrutify.Services/Services/SendEmailQueueService.cs

```diff
@@ -43,5 +44,15 @@ public void SendEmailQueueForTest(IEnumerable<CandidateDTO> candidates, ProjectD
                 BackgroundJob.Enqueue(() => _sendEmailService.SendEmailAsync(emailRequest));
             }
         }
+
+        public void SendEmailQueueForInvite(IEnumerable<Interview> interviews)
+        {
+            var requests = _formEmailService.GetEmailRequestsForInterviewInvite(interviews);
+
+            foreach (var emailRequest in requests)
```

Comment:
Why not to put foreach from all methods to private, not to duplicate ?

Reply:
+